app/models/Product.coffee

```diff
@@ -4,3 +4,15 @@ module.exports = class ProductModel extends CocoModel
   @className: 'Product'
   @schema: require 'schemas/models/product.schema'
   urlRoot: '/db/products'
+
+  priceStringNoSymbol: -> (@get('amount') / 100).toFixed(2)
+
+  purchase: (token, options={}) ->
+    options.url = _.result(@, 'url') + '/purchase'
+    options.method = 'POST'
+    options.data ?= {}
+    options.data.token = token.id
+    options.data.timestamp = new Date().getTime()
+    options.data = JSON.stringify(options.data)
+    options.contentType = 'application/json'
+    return $.ajax(options)
```

Comment:
Yeah... I realized this was a larger issue and I kind of just skipped it for time. I'm tempted, though, to start with a jqxhr wrapper function which makes it behave like a `fetch` Promise, where it resolves no matter what status you get (as long as the server responds at all). Whether bluebird wrapped or not, jquery "promises" reject 400+ status codes. What do you think of that?